web/src/components/Upstream/UpstreamForm.tsx

```diff
@@ -83,6 +83,7 @@ const UpstreamForm: React.FC<Props> = forwardRef(
 
     const resetForm = (upstream_id: string) => {
       if (upstream_id === undefined) {
+        setReadonly(disabled);
```

Comment:
noted, need @guoqqqi to confirm

Reply:
Yes, currently all can be edited on the preview page when first configuring upstream.

Reply:
Does this will affect Route module?

Reply:
> Does this will affect Route module?

Yes！When the router, service, and upstream are created for the first time, this problem also occurs in the upstream involved.